packages/compat/src/empty-package-tree.ts

```diff
@@ -15,6 +15,7 @@ export default class extends Plugin {
   build() {
     if (!this.built) {
       writeJSONSync(join(this.outputPath, 'package.json'), {
+        '//': 'This empty package was created by embroider. If you are having issues resolving this package and you have followed your dependency tree to this file you could be experiencing an issue with your dependencies. See https://github.com/embroider-build/embroider/blob/main/docs/pnpm-injected.md for more information or open an issue in the embroider repo with a recreation of your problem.',
```

Comment:
do folks using yarn or npm have a path to resolve here?

or is it:
- if you're using `yarn@1`. Don't. Use `yarn@3`
- if you're using `npm@9`... ?
- if you're using `pnpm@7`, use `dependenciesMeta.*.injected`

Reply:
Both the PR and this comment are over-focused on package managers here. This comment should not be about package managers at all. There are multiple reasons somebody could have an empty package, and this should only point to a document about that without assuming one of the reasons or assuming what the fix is.

Reply:
So my thoughts are that this comment should be pretty generic and that the thing that it links to should have all the caveats and differences there.

Maybe it's important to make the markdown file not have pnpm in the name then? maybe `docs/injected-dependencies.md`? what do you think?

Reply:
It's not about injected dependencies either. See https://github.com/embroider-build/embroider/pull/1349#issuecomment-1412542979